tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -28,6 +28,18 @@ public TracerSettings()
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TracerSettings"/> class with default values,
```

Comment:
There are several uses of the word "default" in here, perhaps it can be reworded slightly?

Reply:
Agreed, done in https://github.com/DataDog/dd-trace-dotnet/pull/2266/commits/d3d5e3d265ba690cdf9cbda08678624c08d3ed4a